modules/cli/pages/index.adoc

```diff
@@ -7,9 +7,11 @@ include::{mod-loc}attributes.adoc[]
 = Configuring Skupper sites using the CLI
 
 [role="system:abstract"]
-Using the Skupper command-line interface (CLI) allows you to create and manage Skupper sites from the context of the current namespace.
+Using the Skupper command-line interface (CLI) allows you to create and manage {skupper-name} sites from the context of the current namespace.
 
+A typical workflow is to create a site, link sites together, and expose services to the {service-network}.
 
+// tag::installing-cli[]
```

Comment:
The links to the tarballs are not working. See valid links below:

Linux:
https://github.com/skupperproject/skupper/releases/download/0.4.2/skupper-cli-0.4.2-linux-amd64.tgz

MacOS:
https://github.com/skupperproject/skupper/releases/download/0.4.2/skupper-cli-0.4.2-mac-amd64.tgz